zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -274,6 +296,19 @@ private JToggleButton getButtonUseCookies() {
         return useCookies;
     }
 
+    private JToggleButton getButtonUseCsrf() {
+        if (useCsrf == null) {
+            useCsrf =
+                    new JToggleButton(
+                            new ImageIcon(
+                                    HttpPanelSender.class.getResource(
+                                            "/resource/icon/fugue/document-target.png")));
+            useCsrf.setToolTipText(Constant.messages.getString("manReq.checkBox.useCSRF"));
+            useCsrf.setSelected(true);
```

Comment:
Should be disabled by default (other options are).

Reply:
I was trying to follow the implementation of [`getButtonFollowRedirects()`](https://github.com/zaproxy/zaproxy/pull/5828/files/b7e651c9fab6a40e496ef83f6ca6d997d5ded0dd#diff-1a60d73067c4bf50dcb04b02bd283bddR264) which set the default as enabled. But disabled by default makes more sense especially since the button shouldn't even be there unless the `ExtensionAntiCSRF` is enabled.

Reply:
Right, was thinking on the session options.